lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_template.py

```diff
@@ -159,9 +163,13 @@ def update_fields(p):
         v = params.pop(old_k)
         params_update[new_k] = v
 
-    extra_vars = params.get('extra_vars_path')
+    extra_vars = params.get('extra_vars')
     if extra_vars is not None:
-        params_update['extra_vars'] = ['@' + extra_vars]
+        params_update['extra_vars'] = [str(extra_vars)]
+
+    extra_vars_path = params.get('extra_vars_path')
+    if extra_vars_path is not None:
+        params_update['extra_vars'] = ['@' + extra_vars_path]
```

Comment:
might be nice to fail if they are both provided.

Reply:
Good suggestion, Perhaps we can first merge this as this is already reviewed